### PR TITLE
Add NETBIOS name as an input variable

### PR DESCRIPTION
**Pull request by jsf9k**
## 🗣 Description ##

This pull request adds the NETBIOS name as an input variable.

## 💭 Motivation and context ##

This option is now required, even though it isn't actually used in our case.

See also:
- cisagov/ansible-role-freeipa-server#47
- cisagov/freeipa-server-tf-module#59

## 🧪 Testing ##

Automated testing passes. I also used this code to deploy a new staging AMI and verified that it functions as expected.

## ✅ Pre-approval checklist ##

- [x] This PR has an informative and human-readable title.
- [x] Changes are limited to a single goal - _eschew scope creep!_
- [x] All relevant type-of-change labels have been added.
- [x] I have read the [CONTRIBUTING](../blob/develop/CONTRIBUTING.md) document.
- [x] These code changes follow [cisagov code standards](https://github.com/cisagov/development-guide).
- [x] All relevant repo and/or project documentation has been updated to reflect the changes in this PR.
- [x] All new and existing tests pass.

## ✅ Pre-merge checklist ##

- [x] Revert dependencies to default branches.

### CODE REVIEW

**Review by dav3r — APPROVED** [review]
👍

File: README.md [hunk 1]
```diff
@@ -84,6 +84,7 @@ module "example" {
 | nessus\_hostname\_key | The SSM Parameter Store key whose corresponding value contains the hostname of the CDM Tenable Nessus server to which the Nessus Agent should link (e.g. /cdm/nessus/hostname). | `string` | `"/cdm/nessus_hostname"` | no |
 | nessus\_key\_key | The SSM Parameter Store key whose corresponding value contains the secret key that the Nessus Agent should use when linking with the CDM Tenable Nessus server (e.g. /cdm/nessus/key). | `string` | `"/cdm/nessus_key"` | no |
 | nessus\_port\_key | The SSM Parameter Store key whose corresponding value contains the port to which the Nessus Agent should connect when linking with the CDM Tenable Nessus server (e.g. /cdm/nessus/port). | `string` | `"/cdm/nessus_port"` | no |
+| netbios\_name | The NETBIOS name to be used by the server (e.g. EXAMPLE).  Note that NETBIOS names are restricted to at most 15 characters.  These characters must consist only of uppercase letters, numbers, and dashes. | `string` | `"COOL"` | no |
```
Comment (mcdonnnj) [inline README.md:87]: Given the limit should we add something like in https://github.com/cisagov/ssm-read-role-tf-module/pull/19 (and related) to truncate down to 15 characters.?

Comment (mcdonnnj) [inline README.md]:
```suggestion
| netbios\_name | The NetBIOS name to be used by the server (e.g. EXAMPLE).  Note that NetBIOS names are restricted to at most 15 characters.  These characters must consist only of uppercase letters, numbers, and dashes. | `string` | `"COOL"` | no |
```

File: variables.tf [hunk 2]
```diff
@@ -44,6 +44,16 @@ variable "nessus_port_key" {
   default     = "/cdm/nessus_port"
 }
 
+variable "netbios_name" {
+  type        = string
+  description = "The NETBIOS name to be used by the server (e.g. EXAMPLE).  Note that NETBIOS names are restricted to at most 15 characters.  These characters must consist only of uppercase letters, numbers, and dashes."
```
Comment (mcdonnnj) [inline variables.tf]:
```suggestion
  description = "The NetBIOS name to be used by the server (e.g. EXAMPLE).  Note that NetBIOS names are restricted to at most 15 characters.  These characters must consist only of uppercase letters, numbers, and dashes."
```

File: variables.tf [hunk 3]
```diff
@@ -44,6 +44,16 @@ variable "nessus_port_key" {
...
+  default     = "COOL"
+  validation {
+    condition     = length(var.netbios_name) <= 15 && length(regexall("[^A-Z0-9-]", var.netbios_name)) == 0
+    error_message = "NETBIOS names are restricted to at most 15 characters.  These characters must consist only of uppercase letters, numbers, and dashes."
```
Comment (mcdonnnj) [inline variables.tf]:
```suggestion
    error_message = "NetBIOS names are restricted to at most 15 characters.  These characters must consist only of uppercase letters, numbers, and dashes."
```

**Review by mcdonnnj — APPROVED** [review]
Only thing from me is the capitalization of NetBIOS.

**Pull request merged**